src/app/components/SheriffProfileDetailsModal.tsx

```diff
@@ -5,26 +5,26 @@ import {
     Modal, 
     Image
 } from 'react-bootstrap';
-import { Sheriff, BLANK_SHERIFF } from '../api/index';
-import { default as SheriffProfileView } from './SheriffProfileView';
+import { Sheriff, BLANK_SHERIFF } from '../api';
+import SheriffProfileDetails  from './SheriffProfileDetails';
 
-export interface ViewSheriffProfileModalProps{
+export interface SheriffProfileDetailsModalProps{
     open?: boolean;
     sheriff: Sheriff;
 }
 
-export interface ViewSheriffProfileModalState{
+export interface SheriffProfileDetailsModalState{
     showModal?: boolean;
 
 }
 
-export default class ViewSheriffProfileModal extends React.Component<ViewSheriffProfileModalProps, ViewSheriffProfileModalState>{
-    static defaultProps:ViewSheriffProfileModalProps = {
+export default class SheriffProfileDetailsModal extends React.Component<SheriffProfileDetailsModalProps, SheriffProfileDetailsModalState>{
```

Comment:
We could probably create a more generic modal similar to the Selector that we made.
But not a requirement at this moment, would just consolidate the show/close logic if we chose to change things.

Reply:
Good idea - let's keep this in mind for the upcoming development